src/xmalloc.c

```diff
@@ -315,6 +319,7 @@ static uint32_t sha_block[HASH_BLOCK_SIZE];
     #if defined(WOLFBOOT_HASH_SHA256) || defined(WOLFBOOT_HASH_SHA384)
         { (uint8_t *)sha_block, HASH_BLOCK_SIZE * sizeof(uint32_t), 0 },
     #endif
+        { asncheck_buf, ASNCHECK_BUF_SIZE, 0 },
```

Comment:
It appears that this extra buffer is now needed since wolfSSL v5.7.0 due to `CALLOC_ASNGETDATA` in `wc_RsaPublicKeyDecode_ex()`